accepted/future-releases/constructor-tearoffs/feature-specification.md

```diff
@@ -485,42 +548,72 @@ extension Ext on C {
 }
 class D extends C with M {
   void method() {
-    var f4 = super.inst<int>; // works like (int $) => super.inst<int>($)
-    var f4TypeName = super.inst<int>.runtimeType.toString();
+    var f4 = super.inst<int>; // works like (int $) => super.inst<int>($)    
+    var f4TypeName = super.inst<int>.runtimeType.toString();  
   }
 }
 void main() {
   // Type literals.
   var t1 = List<int>; // Type object for `List<int>`.
   var t2 = ListList<int>; // Type object for `List<List<int>>`.
-  
-  // Instantiated function tear-offs.
+  // Instantiated function tear-offs.  
```

Comment:
Personally I perfer the old version of this code block (with the blank lines)

Reply:
Done.